external/lanelet2_extension_psim/docs/lanelet2_format_extension.md

```diff
@@ -5,7 +5,7 @@ About the basics of the default format, please refer to main [Lanelet2 repositor
 In addition to default Lanelet2 Format, users should add following mandatory/optional tags to their osm lanelet files as explained in reset of this document. 
 Users may use `autoware_lanelet2_validation` [node](../README.md#nodes) to check if their maps are valid. 
 
-There are also [extra regulatory elements](./ExtraRegulatoryElements) added for Autoware:
+There are also [extra regulatory elements](./extra regulatory elements.md) added for Autoware:
```

Comment:
Is this modification intentional?

Reply:
Yes.
Both writing methods work, but the writing method before the change gives an error in the CI of LinkCheck.